[gmp] build error on x64-linux

Package: gmp:x64-linux@6.3.0#2

**Host Environment**

- Host: x64-linux
- Compiler: GNU 15.2.1
- CMake Version: 4.1.1
-    vcpkg-tool version: 2025-09-03-4580816534ed8fd9634ac83d46471440edd82dfe
    vcpkg-scripts version: 2e6fcc4457 2025-09-26 (30 hours ago)

**To Reproduce**

`vcpkg install gmp`

**Failure logs**

```
-- Using cached gmp-6.3.0.tar.xz
-- Cleaning sources at /home/rj/.local/share/vcpkg/buildtrees/gmp/src/v6.3.0-27b6a2315b.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/rj/.local/share/vcpkg/downloads/gmp-6.3.0.tar.xz
-- Applying patch asmflags.patch
-- Applying patch cross-tools.patch
-- Applying patch subdirs.patch
-- Applying patch msvc_symbol.patch
-- Applying patch arm64-coff.patch
-- Applying patch remove_compiler_info.patch
-- Using source at /home/rj/.local/share/vcpkg/buildtrees/gmp/src/v6.3.0-27b6a2315b.clean
Disabling long long reliability test in configure.ac
sed: -e expression #1, char 1: unknown command: `''
sed: -e expression #1, char 1: unknown command: `''
CMake Warning at scripts/cmake/vcpkg_replace_string.cmake:17 (message):
  vcpkg_replace_string made no changes.
Call Stack (most recent call first):
  ports/gmp/portfile.cmake:76 (vcpkg_replace_string)
  scripts/ports.cmake:206 (include)


CMake Warning at scripts/cmake/vcpkg_replace_string.cmake:17 (message):
  vcpkg_replace_string made no changes.
Call Stack (most recent call first):
  ports/gmp/portfile.cmake:88 (vcpkg_replace_string)
  scripts/ports.cmake:206 (include)


-- Found external ninja('1.12.1').
-- Getting CMake variables for x64-linux
-- Loading CMake variables from /home/rj/.local/share/vcpkg/buildtrees/gmp/cmake-get-vars_C_CXX-x64-linux.cmake.log
-- Getting CMake variables for x64-linux-dbg
-- Getting CMake variables for x64-linux-rel
-- Generating configure for x64-linux
-- Finished generating configure for x64-linux
-- Configuring x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: /usr/bin/bash -c "V=1 ./../src/v6.3.0-27b6a2315b.clean/configure  \"ac_cv_type_long_long=yes\" \"gmp_cv_func_longlong=yes\" \"gmp_cv_asm_longlong=yes\" \"CCAS=cc\" \"ASMFLAGS=-c\" \"--enable-cxx\" \"--with-pic\" \"--with-readline=no\" \"gmp_cv_prog_exeext_for_build=\" \"CFLAGS=-fPIC -g -Wno-error\" \"CXXFLAGS=-fPIC -g -Wno-error\" \"--disable-silent-rules\" \"--verbose\" \"--disable-shared\" \"--enable-static\" \"--prefix=/home/rj/.local/share/vcpkg/installed/x64-linux/debug\" \"--bindir=\\${prefix}/../tools/gmp/debug/bin\" \"--sbindir=\\${prefix}/../tools/gmp/debug/sbin\" \"--libdir=\\${prefix}/lib\" \"--includedir=\\${prefix}/../include\" \"--datarootdir=\\${prefix}/share/gmp\""
    Working Directory: /home/rj/.local/share/vcpkg/buildtrees/gmp/x64-linux-dbg
    Error code: 2
    See logs for more information:
      /home/rj/.local/share/vcpkg/buildtrees/gmp/config-x64-linux-dbg-config.log
      /home/rj/.local/share/vcpkg/buildtrees/gmp/config-x64-linux-dbg-out.log
      /home/rj/.local/share/vcpkg/buildtrees/gmp/config-x64-linux-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:866 (vcpkg_execute_required_process)
  ports/gmp/portfile.cmake:152 (vcpkg_configure_make)
  scripts/ports.cmake:206 (include)
```

<details><summary>/home/rj/.local/share/vcpkg/buildtrees/gmp/config-x64-linux-dbg-err.log</summary>

```
./../src/v6.3.0-27b6a2315b.clean/configure: line 8013: syntax error near unexpected token `fi'
./../src/v6.3.0-27b6a2315b.clean/configure: line 8013: `        fi'
```
</details>

<details><summary>/home/rj/.local/share/vcpkg/buildtrees/gmp/config-x64-linux-dbg-out.log</summary>

```
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking for a BSD-compatible install... /usr/bin/install -c
checking whether sleep supports fractional seconds... yes
checking filesystem timestamp resolution... 0.01
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking xargs -n works... yes
checking whether UID '1000' is supported by ustar format... yes
checking whether GID '984' is supported by ustar format... yes
checking how to create a ustar tar archive... gnutar
checking whether to enable maintainer-specific portions of Makefiles... no
```
</details>

<details><summary>/home/rj/.local/share/vcpkg/buildtrees/gmp/config-x64-linux-dbg-config.log</summary>

```
This file contains any messages produced by compilers while
running configure, to aid debugging if configure makes a mistake.

It was created by GNU MP configure 6.3.0, which was
generated by GNU Autoconf 2.72.  Invocation command line was

  $ ./../src/v6.3.0-27b6a2315b.clean/configure ac_cv_type_long_long=yes gmp_cv_func_longlong=yes gmp_cv_asm_longlong=yes CCAS=cc ASMFLAGS=-c --enable-cxx --with-pic --with-readline=no gmp_cv_prog_exeext_for_build= 'CFLAGS=-fPIC -g -Wno-error' 'CXXFLAGS=-fPIC -g -Wno-error' --disable-silent-rules --verbose --disable-shared --enable-static --prefix=/home/rj/.local/share/vcpkg/installed/x64-linux/debug '--bindir=${prefix}/../tools/gmp/debug/bin' '--sbindir=${prefix}/../tools/gmp/debug/sbin' '--libdir=${prefix}/lib' '--includedir=${prefix}/../include' '--datarootdir=${prefix}/share/gmp'

## --------- ##
## Platform. ##
## --------- ##

hostname = RJ-WORKSTATION
uname -m = x86_64
uname -r = 6.16.8-arch3-1
uname -s = Linux
uname -v = #1 SMP PREEMPT_DYNAMIC Mon, 22 Sep 2025 22:08:35 +0000

/usr/bin/uname -p = unknown
/bin/uname -X     = unknown

/bin/arch              = unknown
/usr/bin/arch -k       = unknown
/usr/convex/getsysinfo = unknown
/usr/bin/hostinfo      = unknown
/bin/machine           = unknown
/usr/bin/oslevel       = unknown
/bin/universe          = unknown

PATH: /usr/local/bin/
PATH: /usr/bin/
PATH: /usr/local/sbin/
PATH: /var/lib/flatpak/exports/bin/
PATH: /usr/lib/jvm/default/bin/
PATH: /usr/bin/site_perl/
PATH: /usr/bin/vendor_perl/
PATH: /usr/bin/core_perl/
PATH: /usr/bin/
PATH: /usr/bin/
PATH: /usr/bin/
PATH: /bin/


## ----------- ##
## Core tests. ##
## ----------- ##

configure:3437: looking for aux files: ltmain.sh compile missing install-sh config.guess config.sub
configure:3450:  trying ./../src/v6.3.0-27b6a2315b.clean/
configure:3479:   ./../src/v6.3.0-27b6a2315b.clean/ltmain.sh found
configure:3479:   ./../src/v6.3.0-27b6a2315b.clean/compile found
configure:3479:   ./../src/v6.3.0-27b6a2315b.clean/missing found
configure:3461:   ./../src/v6.3.0-27b6a2315b.clean/install-sh found
configure:3479:   ./../src/v6.3.0-27b6a2315b.clean/config.guess found
configure:3479:   ./../src/v6.3.0-27b6a2315b.clean/config.sub found
configure:3640: checking build system type
configure:3656: result: x86_64-pc-linux-gnu
configure:3676: checking host system type
configure:3691: result: x86_64-pc-linux-gnu
configure:3729: checking for a BSD-compatible install
configure:3803: result: /usr/bin/install -c
configure:3814: checking whether sleep supports fractional seconds
configure:3830: result: yes
configure:3833: checking filesystem timestamp resolution
configure:3968: result: 0.01
configure:3973: checking whether build environment is sane
configure:4018: result: yes
configure:4189: checking for a race-free mkdir -p
configure:4232: result: /usr/bin/mkdir -p
configure:4239: checking for gawk
configure:4260: found /usr/bin/gawk
configure:4272: result: gawk
configure:4283: checking whether make sets $(MAKE)
configure:4307: result: yes
configure:4333: checking whether make supports nested variables
configure:4352: result: yes
configure:4366: checking xargs -n works
configure:4382: result: yes
configure:4467: checking whether UID '1000' is supported by ustar format
configure:4473: result: yes
configure:4480: checking whether GID '984' is supported by ustar format
configure:4486: result: yes
configure:4494: checking how to create a ustar tar archive
configure:4505: tar --version
tar (GNU tar) 1.35
Copyright (C) 2023 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
configure:4508: $? = 0
configure:4548: tardir=conftest.dir && eval tar --format=ustar -chf - "$tardir" >conftest.tar
configure:4551: $? = 0
configure:4555: tar -xf - <conftest.tar
configure:4558: $? = 0
configure:4560: cat conftest.dir/file
GrepMe
configure:4563: $? = 0
configure:4578: result: gnutar
configure:4607: checking whether to enable maintainer-specific portions of Makefiles
configure:4618: result: no
User:
ABI=
CC=/usr/bin/cc
CFLAGS=-fPIC -g -Wno-error
CPPFLAGS=(unset)
MPN_PATH=
GMP:
abilist=64 x32 32
cclist=gcc icc cc

## ---------------- ##
## Cache variables. ##
## ---------------- ##

ac_cv_build=x86_64-pc-linux-gnu
ac_cv_env_ABI_set=
ac_cv_env_ABI_value=
ac_cv_env_CCC_set=
ac_cv_env_CCC_value=
ac_cv_env_CC_FOR_BUILD_set=set
ac_cv_env_CC_FOR_BUILD_value=/usr/bin/cc
ac_cv_env_CC_set=set
ac_cv_env_CC_value=/usr/bin/cc
ac_cv_env_CFLAGS_set=set
ac_cv_env_CFLAGS_value='-fPIC -g -Wno-error'
ac_cv_env_CPPFLAGS_set=
ac_cv_env_CPPFLAGS_value=
ac_cv_env_CPP_FOR_BUILD_set=set
ac_cv_env_CPP_FOR_BUILD_value='/usr/bin/cc -E'
ac_cv_env_CPP_set=
ac_cv_env_CPP_value=
ac_cv_env_CXXCPP_set=
ac_cv_env_CXXCPP_value=
ac_cv_env_CXXFLAGS_set=set
ac_cv_env_CXXFLAGS_value='-fPIC -g -Wno-error'
ac_cv_env_CXX_set=set
ac_cv_env_CXX_value=/usr/bin/c++
ac_cv_env_LDFLAGS_set=
ac_cv_env_LDFLAGS_value=
ac_cv_env_LIBS_set=
ac_cv_env_LIBS_value=
ac_cv_env_LT_SYS_LIBRARY_PATH_set=
ac_cv_env_LT_SYS_LIBRARY_PATH_value=
ac_cv_env_M4_set=
ac_cv_env_M4_value=
ac_cv_env_YACC_set=
ac_cv_env_YACC_value=
ac_cv_env_YFLAGS_set=
ac_cv_env_YFLAGS_value=
ac_cv_env_build_alias_set=
ac_cv_env_build_alias_value=
ac_cv_env_host_alias_set=
ac_cv_env_host_alias_value=
ac_cv_env_target_alias_set=
ac_cv_env_target_alias_value=
ac_cv_host=x86_64-pc-linux-gnu
ac_cv_path_install='/usr/bin/install -c'
ac_cv_path_mkdir=/usr/bin/mkdir
ac_cv_prog_AWK=gawk
ac_cv_prog_make_make_set=yes
ac_cv_type_long_long=yes
am_cv_filesystem_timestamp_resolution=0.01
am_cv_make_support_nested_variables=yes
am_cv_prog_tar_ustar=gnutar
am_cv_sleep_fractional_seconds=yes
am_cv_xargs_n_works=yes
gmp_cv_asm_longlong=yes
gmp_cv_func_longlong=yes
gmp_cv_prog_exeext_for_build=

## ----------------- ##
## Output variables. ##
## ----------------- ##

ABI=''
ACLOCAL='${SHELL} '\''/home/rj/.local/share/vcpkg/buildtrees/gmp/src/v6.3.0-27b6a2315b.clean/missing'\'' aclocal-1.18'
AMTAR='$${TAR-tar}'
AM_BACKSLASH='\'
AM_DEFAULT_V=''
AM_DEFAULT_VERBOSITY='1'
AM_V=''
AR='/usr/bin/ar'
AS=''
ASMFLAGS='-c'
AUTOCONF='${SHELL} '\''/home/rj/.local/share/vcpkg/buildtrees/gmp/src/v6.3.0-27b6a2315b.clean/missing'\'' autoconf'
AUTOHEADER='${SHELL} '\''/home/rj/.local/share/vcpkg/buildtrees/gmp/src/v6.3.0-27b6a2315b.clean/missing'\'' autoheader'
AUTOMAKE='${SHELL} '\''/home/rj/.local/share/vcpkg/buildtrees/gmp/src/v6.3.0-27b6a2315b.clean/missing'\'' automake-1.18'
AWK='gawk'
CALLING_CONVENTIONS_OBJS='x86call.lo x86check$U.lo'
CC='/usr/bin/cc'
CCAS='cc'
CC_FOR_BUILD='/usr/bin/cc'
CFLAGS='-fPIC -g -Wno-error'
CPP=''
CPPFLAGS=''
CPP_FOR_BUILD='/usr/bin/cc -E'
CSCOPE='cscope'
CTAGS='ctags'
CXX='/usr/bin/c++'
CXXCPP=''
CXXFLAGS='-fPIC -g -Wno-error'
CYGPATH_W='echo'
DEFN_LONG_LONG_LIMB=''
DEFS=''
DLLTOOL=''
DSYMUTIL=''
DUMPBIN=''
ECHO_C=''
ECHO_N='-n'
ECHO_T=''
EGREP=''
ENABLE_STATIC_FALSE=''
ENABLE_STATIC_TRUE=''
ETAGS='etags'
EXEEXT=''
EXEEXT_FOR_BUILD=''
FGREP=''
FILECMD=''
GMP_LDFLAGS=''
GMP_LIMB_BITS=''
GMP_NAIL_BITS='0'
GREP=''
HAVE_CLOCK_01=''
HAVE_CPUTIME_01=''
HAVE_GETRUSAGE_01=''
HAVE_GETTIMEOFDAY_01=''
HAVE_HOST_CPU_FAMILY_power='0'
HAVE_HOST_CPU_FAMILY_powerpc='0'
HAVE_SIGACTION_01=''
HAVE_SIGALTSTACK_01=''
HAVE_SIGSTACK_01=''
HAVE_STACK_T_01=''
HAVE_SYS_RESOURCE_H_01=''
INSTALL_DATA='${INSTALL} -m 644'
INSTALL_PROGRAM='${INSTALL}'
INSTALL_SCRIPT='${INSTALL}'
INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
LD='/usr/bin/ld'
LDFLAGS=''
LEX=''
LEXLIB=''
LEX_OUTPUT_ROOT=''
LIBCURSES=''
LIBGMPXX_LDFLAGS=''
LIBGMP_DLL=''
LIBGMP_LDFLAGS=''
LIBM=''
LIBM_FOR_BUILD=''
LIBOBJS=''
LIBREADLINE=''
LIBS=''
LIBTOOL=''
LIPO=''
LN_S=''
LTLIBOBJS=''
LT_SYS_LIBRARY_PATH=''
M4=''
MAINT='#'
MAINTAINER_MODE_FALSE=''
MAINTAINER_MODE_TRUE='#'
MAKEINFO='${SHELL} '\''/home/rj/.local/share/vcpkg/buildtrees/gmp/src/v6.3.0-27b6a2315b.clean/missing'\'' makeinfo'
MANIFEST_TOOL=''
MKDIR_P='/usr/bin/mkdir -p'
NM='/usr/bin/nm'
NMEDIT=''
OBJDUMP='/usr/bin/objdump'
OBJEXT=''
OTOOL64=''
OTOOL=''
PACKAGE='gmp'
PACKAGE_BUGREPORT='gmp-bugs@gmplib.org (see https://gmplib.org/manual/Reporting-Bugs.html)'
PACKAGE_NAME='GNU MP'
PACKAGE_STRING='GNU MP 6.3.0'
PACKAGE_TARNAME='gmp'
PACKAGE_URL='https://www.gnu.org/software/gmp/'
PACKAGE_VERSION='6.3.0'
PATH_SEPARATOR=':'
RANLIB='/usr/bin/ranlib'
SED=''
SET_MAKE=''
SHELL='/bin/sh'
SPEED_CYCLECOUNTER_OBJ='pentium.lo'
STRIP='/usr/bin/strip'
TAL_OBJECT=''
TUNE_LIBS=''
TUNE_SQR_OBJ=''
U_FOR_BUILD=''
VERSION='6.3.0'
WANT_CXX_FALSE=''
WANT_CXX_TRUE=''
WITH_READLINE_01=''
YACC=''
YFLAGS=''
ac_ct_AR=''
ac_ct_CC=''
ac_ct_CXX=''
ac_ct_DUMPBIN=''
am__EXEEXT_FALSE=''
am__EXEEXT_TRUE=''
am__isrc=' -I$(srcdir)'
am__leading_dot='.'
am__rm_f_notfound=''
am__tar='tar --format=ustar -chf - "$$tardir"'
am__untar='tar -xf -'
am__xargs_n='xargs -n'
bindir='${prefix}/../tools/gmp/debug/bin'
build='x86_64-pc-linux-gnu'
build_alias=''
build_cpu='x86_64'
build_os='linux-gnu'
build_vendor='pc'
datadir='${datarootdir}'
datarootdir='${prefix}/share/gmp'
docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
dvidir='${docdir}'
exec_prefix='NONE'
gmp_srclinks=''
host='x86_64-pc-linux-gnu'
host_alias=''
host_cpu='x86_64'
host_os='linux-gnu'
host_vendor='pc'
htmldir='${docdir}'
includedir='${prefix}/../include'
infodir='${datarootdir}/info'
install_sh='${SHELL} /home/rj/.local/share/vcpkg/buildtrees/gmp/src/v6.3.0-27b6a2315b.clean/install-sh'
libdir='${prefix}/lib'
libexecdir='${exec_prefix}/libexec'
localedir='${datarootdir}/locale'
localstatedir='${prefix}/var'
mandir='${datarootdir}/man'
mkdir_p='$(MKDIR_P)'
mpn_objects=''
mpn_objs_in_libgmp=''
oldincludedir='/usr/include'
pdfdir='${docdir}'
prefix='/home/rj/.local/share/vcpkg/installed/x64-linux/debug'
program_transform_name='s,x,x,'
psdir='${docdir}'
runstatedir='${localstatedir}/run'
sbindir='${prefix}/../tools/gmp/debug/sbin'
sharedstatedir='${prefix}/com'
sysconfdir='${prefix}/etc'
target_alias=''

## ----------- ##
## confdefs.h. ##
## ----------- ##

/* confdefs.h */
#define PACKAGE_NAME "GNU MP"
#define PACKAGE_TARNAME "gmp"
#define PACKAGE_VERSION "6.3.0"
#define PACKAGE_STRING "GNU MP 6.3.0"
#define PACKAGE_BUGREPORT "gmp-bugs@gmplib.org (see https://gmplib.org/manual/Reporting-Bugs.html)"
#define PACKAGE_URL "https://www.gnu.org/software/gmp/"
#define PACKAGE "gmp"
#define VERSION "6.3.0"
#define WANT_FFT 1
#define HAVE_HOST_CPU_x86_64 1

configure: exit 2
```
</details>

The gmp in vcpkg is missing changeset `8e7bb4ae7a18`, which fixes a bug in BMI2 detection within the conftest.

```diff
changeset:   18477:8e7bb4ae7a18
user:        Marc Glisse <marc.glisse@inria.fr>
date:        Wed Jan 29 22:38:02 2025 +0100
summary:     Complete function prototype in acinclude.m4 for C23 compatibility

diff -r e84c5c785bbe -r 8e7bb4ae7a18 ChangeLog
--- a/ChangeLog    Wed Dec 04 18:26:27 2024 +0100
+++ b/ChangeLog    Wed Jan 29 22:38:02 2025 +0100
@@ -1,3 +1,8 @@
+2025-01-29  Rudi Heitbaum <rudi@heitbaum.com>
+        Marc Glisse <marc.glisse@inria.fr>
+
+    * acinclude.m4: Complete function prototype.
+
 2024-05-15 Sven Anderson <sven@anderson.de>
 
     * mpn/x86_64/fat/fat.c: Fix bug in BMI2 detection.
diff -r e84c5c785bbe -r 8e7bb4ae7a18 acinclude.m4
--- a/acinclude.m4    Wed Dec 04 18:26:27 2024 +0100
+++ b/acinclude.m4    Wed Jan 29 22:38:02 2025 +0100
@@ -609,7 +609,7 @@
 
 #if defined (__GNUC__) && ! defined (__cplusplus)
 typedef unsigned long long t1;typedef t1*t2;
-void g(){}
+void g(int,t1 const*,t1,t2,t1 const*,int){}
 void h(){}
 static __inline__ t1 e(t2 rp,t2 up,int n,t1 v0)
 {t1 c,x,r;int i;if(v0){c=1;for(i=1;i<n;i++){x=up[i];r=x+1;rp[i]=r;}}return c;}```

--- a/ports/gmp/c23.patch
+++ b/ports/gmp/c23.patch
@@ -1,0 +1,37 @@
+diff --git a/acinclude.m4 b/acinclude.m4
+index e84c5c785..8e7bb4ae7 100644
+--- a/acinclude.m4
++++ b/acinclude.m4
+@@ -564,23 +564,6 @@
+ }
+ ])
+ 
+-# __builtin_alloca is not available everywhere, check it exists before
+-# seeing that it works
+-GMP_PROG_CC_WORKS_PART_TEST([$1],[__builtin_alloca availability],
+-[int k; int foo () { __builtin_alloca (k); }],
+-  [GMP_PROG_CC_WORKS_PART([$1], [alloca array],
+-[/* The following provokes an internal compiler error from Itanium HP-UX cc
+-    under +O2 or higher.  We use this sort of code in mpn/generic/mul_fft.c. */
+-int k;
+-int foo ()
+-{
+-  int i, **a;
+-  a = __builtin_alloca (k);
+-  for (i = 0; i <= k; i++)
+-    a[i] = __builtin_alloca (1 << i);
+-}
+-])])
+-
+ GMP_PROG_CC_WORKS_PART([$1], [abs int -> double conversion],
+ [/* The following provokes an internal error from the assembler on
+    power2-ibm-aix4.3.1.0.  gcc -mrios2 compiles to nabs+fcirz, and this
+@@ -609,7 +609,7 @@
+ 
+ #if defined (__GNUC__) && ! defined (__cplusplus)
+ typedef unsigned long long t1;typedef t1*t2;
+-void g(){}
++void g(int,t1 const*,t1,t2,t1 const*,int){}
+ void h(){}
+ static __inline__ t1 e(t2 rp,t2 up,int n,t1 v0)
+ {t1 c,x,r;int i;if(v0){c=1;for(i=1;i<n;i++){x=up[i];r=x+1;rp[i]=r;}}return c;}

--- a/ports/gmp/portfile.cmake
+++ b/ports/gmp/portfile.cmake
@@ -22,6 +22,7 @@ vcpkg_extract_source_archive(SOURCE_PATH
         msvc_symbol.patch
         arm64-coff.patch
         remove_compiler_info.patch
+        c23.patch
 )
 
 vcpkg_list(SET OPTIONS)

--- a/ports/gmp/vcpkg.json
+++ b/ports/gmp/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "gmp",
   "version": "6.3.0",
-  "port-version": 2,
+  "port-version": 3,
   "description": "The GNU Multiple Precision Arithmetic Library",
   "homepage": "https://gmplib.org",
   "license": "LGPL-3.0-only OR GPL-2.0-only",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3374,7 +3374,7 @@
     },
     "gmp": {
       "baseline": "6.3.0",
-      "port-version": 2
+      "port-version": 3
     },
     "gmsh": {
       "baseline": "4.14.0",

--- a/versions/g-/gmp.json
+++ b/versions/g-/gmp.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "9b979ff042700c317cb1ae4f2471e00bcfcf7d82",
+      "version": "6.3.0",
+      "port-version": 3
+    },
+    {
       "git-tree": "d9ef9fbb509e23fcb49f567f3c383ca3fd37e2c0",
       "version": "6.3.0",
       "port-version": 2

